metashare/api/jobs.py

```diff
@@ -208,6 +208,27 @@ def create_branches_on_github_then_create_scratch_org(*, scratch_org):
 )
 
 
+def refresh_scratch_org(scratch_org):
+    user = scratch_org.owner
+    repo_id = scratch_org.task.project.repository.get_repo_id(user)
+    commit_ish = scratch_org.task.branch_name
+
+    delete_org(scratch_org)
+
+    with local_github_checkout(user, repo_id, commit_ish) as repo_root:
+        _create_org_and_run_flow(
+            scratch_org,
+            user=user,
+            repo_id=repo_id,
+            repo_branch=commit_ish,
+            project_path=repo_root,
+        )
```

Comment:
@wlonk Because we're not clearing the scheduled `alert_user_about_expiring_org` job when orgs are deleted, I _think_ this will result in two different jobs for the same org, one with the old expiration date and one with the new one.

Reply:
@wlonk Bump on this.